engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/EngineErrorHandlingTest.java

```diff
@@ -40,7 +75,10 @@
 public final class EngineErrorHandlingTest {
 
   private static final String STREAM_NAME = "foo";
+  private static final ProcessInstanceRecord PROCESS_INSTANCE_RECORD = Records.processInstance(1);
+  private static final JobRecord JOB_RECORD = Records.job(1);
 
+  protected SynchronousLogStream stream;
```

Comment:
:wrench: can be removed

Reply:
Argh seem to come back due to resolving merge conflicts sorry.